packages/strapi/lib/Strapi.js

```diff
@@ -249,9 +249,33 @@ class Strapi {
         }
       }
 
-      // Emit started event.
+      // Get database clients
       const databaseClients = _.map(this.config.get('connections'), _.property('settings.client'));
-      await this.telemetry.send('didStartServer', { database: databaseClients });
+
+      // Get plugins
+      const plugins = { strapi: [], external: [] };
+      for (const plugin in this.plugins) {
+        if (this.plugins[plugin].package.author.name === 'Strapi team') {
```

Comment:
Checking hte package info is not going to be a safe info. 
When loading the plugins you should add a flag to say they are local or not but all strapi-plugin-xxx are external plugins even if not maintained by us. We should just ignore local plugins for now